adage/adage/urls.py

```diff
@@ -10,13 +10,15 @@
     SignatureViewSet,
 )
 from genes.views import GeneViewSet
+from organisms.views import OrganismViewSet
 
 router = routers.DefaultRouter()
 router.register(r"experiments", ExperimentViewSet, basename="experiment")
-router.register(r"mlmodels", MLModelViewSet)
+router.register(r"genes", GeneViewSet, basename="gene")
+router.register(r"models", MLModelViewSet)
+router.register(r"organisms", OrganismViewSet)
 router.register(r"samples", SampleViewSet)
 router.register(r"signatures", SignatureViewSet)
-router.register(r"genes", GeneViewSet, basename="gene")
```

Comment:
What's with all the `r"strings"` (raw string)... these strings don't and won't have backslashes?

Reply:
Fair question. I have it mainly because the Django rest framework documentation always uses raw strings when registering in the router (probably in case sometime later the string may include `\` characters).

Reply:
Okay then fine to keep